release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -50,6 +50,20 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-13-installation-and-upgrade"]
 === Installation and upgrade
 
+[id="ocp-4-13-installation-vsphere-8-support"]
+==== Support for VMware vSphere version 8.0 
+{product-title} {product-version} supports VMware vSphere version 8.0. You can continue to install an {product-title} cluster on VMware vSphere version 7.0 Update 2.
+
+[id="ocp-4-13-installation-vsphere-region-zone"]
+==== VMware vSphere region and zone enablement
+You can deploy an {product-title} cluster to multiple vSphere datacenters, or regions, that run in a single VMware vCenter. Each datacenter can run multiple clusters, or zones. This configuration reduces the risk of a hardware failure or network outage causing your cluster to fail.
+
+You can also specify multiple regions and zones on a cluster after installation. You must upgrade your existing cluster to {product-title} {product-version} before you configure a multiple region and zone environment.
+
+[id="ocp-4-13-installation-vsphere-default-config-yaml"]
+==== Changes to the default `install-config.yaml` file
+After you run the installation program for installing {product-title} on vSphere, the default `install-config.yaml` file now includes `vcenters` and `failureDomains` fields, so that you can choose to specify multiple datacenters, region, and zone information for your cluster. You can leave these fields blank if you want to install an {product-title} cluster in a vSphere environment that consists of single datacenter running in a VMware vCenter. For more information.
```

Comment:
`For more information.` it looks like half of a sentence, do we miss something here?